src/modules/controllers/task.controller.js

```diff
@@ -0,0 +1,34 @@
+const Task = require('../../db/models/task');
+
+module.exports.getAllTasks = async (req, res, next) => {
+  Task.find().then(result => {
+    res.send({data: result});
+  });
+};
+
+module.exports.createNewTask = (req, res, next) => {
+  const task = new Task(req.body);
+  task.save().then(result => {
+    res.send(result);
+  }).catch(err => console.log(err));
+};
+
+module.exports.changeTaskInfo = (req, res, next) => {
```

Comment:
split into two requests. One changes the text, the other changes the checkbox. At your front, they can change separately